### DISCUSSION TITLE
Avoir le PIV sur fond blanc

### DISCUSSION THREAD

**Discussion by cotal03** [MTESSDev/utd-webcomponents#50] [Idées]
Présentement, il y a des sites qui utilisent le PIV sur fond blanc. Est-ce possible de l'ajouter dans les options?

**Comment by j0ber** [comment]
Bonjour,

Nous avons vérifié auprès de l'équipe du système de design Québec.ca.

Voici la réponse officielle que nous avons obtenue :  

> Bonjour,
> 
> Le bandeau blanc fait toujours partie du Programme d’identification visuelle du gouvernement du Québec, régit par le SCT. Toutefois, dans un souci de cohérence dans l’ensemble des sites gouvernementaux, il est fortement conseillé d’utiliser le bandeau PIV bleu. C’est pourquoi c’est cette version seulement qui est documentée dans le Système de design gouvernemental.
> 
> Si vous souhaitez tout de même utiliser le bandeau blanc, vous pouvez communiquer avec l’équipe PIV pour les questions techniques par rapport au PIV blanc. [piv@mce-sct.gouv.qc.ca](mailto:piv@mce-sct.gouv.qc.ca).
> 
> Merci et bonne journée!
> 

Le PIV fond blanc n'est pas recommandé.

Nous vous suggérons donc fortement d'utiliser le PIV avec fond bleu, et n'avons pas l'intention de mettre en place le PIV avec fond blanc afin de respecter le plus possible les règles du  SDG.

En ayant cette réponse officielle de l'équipe de design Québec.ca, il devrait être facile pour vous de "pousser" le PIV fond bleu à vos clients.

Bonne journée!
